src/components/chartRepo/ChartRepo.tsx

```diff
@@ -279,15 +398,34 @@ function ChartForm({ id = null, name = "", active = false, url = "", authMode =
             {state.auth.value === "ACCESS_TOKEN" && <div className="form__row">
                 <ProtectedInput value={customState.accessToken.value} onChange={customHandleChange} name="accessToken" error={customState.accessToken.error} label="Access token*" />
             </div>} */}
+            {chartRepoType !== 'PUBLIC' ? (
+                <Checkbox
+                    rootClassName="fs-13 dc__hover-n50 pt-8 pb-8 pl-8 ml-8"
+                    isChecked={secureWithTls === true}
+                    value={'CHECKED'}
```

Comment:
Use existing check type

Reply:
Also change to `&&` pattern. No need to provide `<></>` as optional.

Reply:
done